### PR TITLE
Rename MerchantToken to AccessToken

### PR DESCRIPTION
**Pull request by aait**

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -21,12 +21,12 @@
 
 // phpcs:disable
 if (getenv('MERCHANT_TOKEN') && getenv('PAYEE_ID')) {
```
Comment (asbjornu) [inline tests/bootstrap.php]:
```suggestion
if (getenv('ACCESS_TOKEN') && getenv('PAYEE_ID')) {
```

File: tests/bootstrap.php [hunk 2]
```diff
@@ -21,12 +21,12 @@
...
-    define('MERCHANT_TOKEN', getenv('MERCHANT_TOKEN'));
+    define('ACCESS_TOKEN', getenv('MERCHANT_TOKEN'));
```
Comment (asbjornu) [inline tests/bootstrap.php]:
```suggestion
    define('ACCESS_TOKEN', getenv('ACCESS_TOKEN'));
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by asbjornu — APPROVED** [review]
This is a breaking change requiring a major bump, right?

**Comment by aait** [comment]
> This is a breaking change requiring a major bump, right?

Yes

**Pull request merged**